components/application-gateway/README.md

```diff
@@ -90,3 +90,4 @@ The Console Backend Service has the following obligatory requirements:
 ### Contribution
 
 To learn how you can contribute to this project, see the [Contributing](/CONTRIBUTING.md) document.
```

Comment:
How did this file end up in this PR?

Reply:
It is done to force rebuilding the component to unlock the upgrade job.